internal/test/test.go

```diff
@@ -498,8 +498,21 @@ func (h *Helper) TempDir(path string) {
 }
 
 // Path returns the absolute pathname to file with the temporary
-// directory.
+// directory. Symlinks are evaluated.
 func (h *Helper) Path(name string) string {
+	path := h.OriginalPath(name)
```

Comment:
Can we instead keep `Path` as-is, and make the new function `EvalPath` or `ResolvedPath` the one that evaluates symlinks?

Reply:
@ibrasho yes, it makes sense. But I'd propose to delay that change after @sdboyer decides if this PR is mergeable or not. Because, this change will just increase the diff by a few hundred lines and increase the possibility of merge conflicts with the current `origin/master`.

If everything is OK, I'll make sure that the PR with this additional change merges in `origin/master` without conflicts.

Reply:
@sdboyer What's your opinion?